scripts/exregional_run_analysis.sh

```diff
@@ -643,7 +695,130 @@ if [ $netcdf_diag = ".true." ]; then
 fi
 
 done
+#
```

Comment:
suggest to submit nMembers GSI jobs and not to do observer for nMembers in one GSI job.

i.e. nMembers GSI metatasks